types/react-subscription.d.ts

```diff
@@ -1,9 +1,9 @@
 
 import Subscription = require('./subscription');
```

Comment:
Should not we switch `require` to `import` here as well?

Reply:
Possible. For consistency. Whatever you say)

Reply:
let's do it!